borgmatic/tests/unit/borg/test_create.py

```diff
@@ -128,15 +118,13 @@ def test_make_exclude_flags_is_empty_when_config_has_no_excludes():
     assert exclude_flags == ()
 
 
-CREATE_COMMAND = ('borg', 'create', 'repo::host-now', 'foo', 'bar')
+CREATE_COMMAND = ('borg', 'create', 'repo::{hostname}-{now:%Y-%m-%dT%H:%M:%S.%f}', 'foo', 'bar')
```

Comment:
To avoid duplication of the default archive name, you could pull ` default_archive_name` out into a constant, and then reference/interpolate it here. I generally try not to refer to code-under-test in tests themselves, but I make an exception for constants. Same thing for other uses of that archive name format below.

I don't feel super strongly about this particular instance of duplication though.